lib/src/react16_suggestors/react_dom_render_migrator.dart

```diff
@@ -70,15 +88,35 @@ class ReactDomRenderMigrator extends GeneralizingAstVisitor
       return ancestor is FunctionDeclaration;
     });
 
-    void addOverReactPatch(int offset) {
-      yieldPatch(
-          offset, offset, 'import \'package:over_react/over_react.dart\';\n');
+    void addOverReactPatch([int offset]) {
+      if (offset == null && imports.isNotEmpty) {
+        offset = imports.last.offset;
+      } else if (libraryDirective.isNotEmpty) {
+        offset = libraryDirective.first.end;
+      } else {
+        offset = 0;
+      }
+      if (!isPartOf) {
+        yieldPatch(
+            offset, offset, 'import \'package:over_react/over_react.dart\';\n');
+      }
+    }
+
+    bool isWrappedWithErrorBoundary = false;
```

Comment:
#supernit

I just now noticed how many variable declarations are occurring throughout the method. I always learned and liked keeping as many variable declarations located as close to the top of the scope as possible, but I can't find a direct Dart best practice for that so it may just be me 🤷‍♂ 

If that seems like something you'd want to clean up, then I was thinking just grouping as many of the simple declarations as possible close to the top and letting the more complex ones remain at the bottom.